### ISSUE TITLE
Docking Connectors at Vesta are not working as of 1:03am 06/04/2024.

### ISSUE THREAD

**Issue by TAGur1t** [DraconisCluster/DX-Issues#44]
** tl; dr **
Docking connectors do not change color and do not engage when docking, though the parking indicator was ticking over saying I was docked and had a 10 min cooldown. Not showing connected grid while accessing trade stations. 

** Full info **
Attempted to dock to Connector Passageway @ Vesta station with a large freighter. In aligning with the connector, I was "magnetized" to the docking port, but go no color change on the passageway. Executed the following steps in trying to resolve the issue:

1. Fixshipped the hauler.
2. ran !entities refresh.
3. F5'ed.
4. As admin, transferred grid to myself, fixship and transferred back.
5. Cycled the connectors, disabled the magnetism and re-enabled. 
6. BFJ restarted DX2. 

All of the above actions had no effect on the outcome of being able to dock/undock to Vesta station. Another player in an different faction was also at Vesta and unable to dock/undock appropriately. 

**Screenshots**
--If applicable, add screenshots to help explain your problem.--

**Comment by TAGur1t** [comment]
possible contributing factor, grid in question is 166 KT.

**Comment by draconb** [comment]
I was able to use a few of the connectors just fine, can you take a picture of which one wasn't working? @TAGur1t

**Comment by TAGur1t** [comment]
were you using a "heavy" ship? 
see picture below of circled connectors that I attempted with the ship. 
![image](https://github.com/DraconisCluster/DX-Issues/assets/85374737/c91f3bb2-d03a-41dc-9a9a-443f70e18cb7)

**Comment by draconb** [comment]
Not 100% sure how heavy, was loaded with mag, uranium, silver and some others iirc, I used the one in the top right of the circle, not the razerback one but the other one in that circle.

**Comment by TAGur1t** [comment]
I would recommend doing it with a ship over 75 KT to see if it triggers anything or not. however, I understand the "problem" being that connectors do have a weight limit, so its just something I need to compensate for.

**Comment by draconb** [comment]
Pretty sure issue was resolved, if not, re-open
